expressEngine/lib/service/db.js

```diff
@@ -1,27 +1,26 @@
-class DbOperation {
-  count = (collection, data = {}) => {
-    return collection.countDocuments(data);
-  }
+import BaseService from './base';
 
-  create = (collection, data = {}) => {
-    return collection.create(data);
-  }
+const baseService = new BaseService();
+class DBOperation {
+  count = (collection, data = {}) => collection.countDocuments(data)
 
-  find = (collection, data = {}, projection = {}) => {
-    return collection.find(data, projection);
-  }
+  create = (collection, data = {}) => collection.create(data)
 
-  findOne = (collection, data = {}, projection = {}) => {
-    return collection.findOne(data, projection);
-  }
+  find = (collection, data = {}, projection = {}) => collection.find(data, projection)
 
-  deleteMany = (collection, data) => {
-    return collection.deleteMany(data);
-  }
+  findOne = (collection, data = {}, projection = {}) => collection.findOne(data, projection)
 
-  deleteOne = (collection, data) => {
-    return collection.deleteOne(data);
+  findAll = ({
+    collection, data = {}, skip = 0, limit = 10, projection = {},
+  }) => {
+    const skipValue = baseService.parseNumber(skip, 0);
```

Comment:
What will happen if I call this function without passing the collection?
Error: undefined.find is not a function.
Please add it's check